README.md

```diff
@@ -316,6 +316,12 @@ scan-frequency = "5m0s"
 #  memory usage (by setting both trie-index and trigram-index to true).
 trie-index = false
 
+# This provides the ability to query for new metrics without any wsp files
+# i.e query for metrics present only in cache. Does a cache-scan and
+# populates index with metrics with or without corresponding wsp files.
+# Disabled by default.
+cache-scan = false
```

Comment:
> but will lead to increased memory consumption

Need to patch here as well.

Reply:
Thank you for mentioning this. Missed to add the comment here. Pushed the change!